src/classes/testAccessController.cls

```diff
@@ -38,6 +38,44 @@
 @isTest
 private class testAccessController {
 
+	// Baseline Contact fields that will be accessiable when using the ReadOnly profile and createFieldPermissions()
+	private static List<String> fieldsToCheck = new List<String>{'LastName','FirstName','Salutation','OtherStreet','OtherCity','OtherState','OtherPostalCode','OtherCountry','OtherLatitude','OtherLongitude','MailingStreet','MailingCity','MailingState','MailingPostalCode','MailingCountry','MailingLatitude','MailingLongitude','Phone','Fax','MobilePhone','HomePhone','OtherPhone','AssistantPhone','ReportsToId','Title','Department','AssistantName','LeadSource','Birthdate','Description','OwnerId','EmailBouncedReason','EmailBouncedDate','Jigsaw','CleanStatus', 'Email'};
```

Comment:
Jigsaw fields might go away with the data.com end of life. Please choose another standard field with a more certain future.

Reply:
I've removed the Jigsaw and CleanStatus data.com fields. They were mostly present as the ReadOnly profile granted access and I was trying to stress the methods under testing with the largest number of fields that standard profile allows.